src/test/java/org/xstefank/check/SkipCheckTest.java

```diff
@@ -0,0 +1,114 @@
+package org.xstefank.check;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.powermock.api.mockito.PowerMockito;
+import org.xstefank.api.GitHubAPI;
+import org.xstefank.model.yaml.FormatConfig;
+import org.xstefank.model.yaml.SkipPatterns;
+import java.io.IOException;
+import java.util.regex.Pattern;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import static org.powermock.api.support.membermodification.MemberMatcher.method;
+import static org.powermock.api.support.membermodification.MemberModifier.suppress;
+import static org.xstefank.check.AuxiliaryTestMethods.helpSetUpSkipPatterns;
+import static org.xstefank.check.AuxiliaryTestMethods.readJsonNode;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest(GitHubAPI.class)
+public class SkipCheckTest {
+
+    private static JsonNode testPullRequestPayload;
+    private static JsonNode testCommitsPayload;
+    private static FormatConfig testFormatConfig = new FormatConfig();
+    private SkipPatterns testSkipPatterns = new SkipPatterns();
+    private static final String stubMethodName = "getJsonWithCommits";
```

Comment:
there are still at least 2 places in these tests that will need to be changed in case this method is renamed